### DISCUSSION TITLE
768 filter/model filter for embedding

### DISCUSSION THREAD

**Discussion by StaffanJOlsson** [civitai/civitai#100] [Ideas]
Filter for Embeddings/Textual Inversions

Now that embeddings are being more used I wish I could browse embedding that supports 768 easier.
This would be a great filter in my opinion. Right now I have to assume if it doesn't say 768 or 2.0/2.1 it won't work on 768.
This would of course require the uploader to infill the information when uploading the embedding.

**Comment by JustMaier** [comment]
**This is live!**

Now when uploading a model you can specify the base model for the version:
![image](https://user-images.githubusercontent.com/607609/208995060-27a4be2f-093c-435a-9ee5-b5821e4a73a4.png)

And we've added the ability to filter based on that data:
![image](https://user-images.githubusercontent.com/607609/208995116-7303dd92-40d8-48a8-816c-6313e350d5f2.png)
